CONTRIBUTING.md

```diff
@@ -90,6 +90,8 @@ Flask-Appbuilder itself comes bundled with jQuery and bootstrap.
 While these may be phased out over time, these packages are currently not
 managed with npm.
 
+### Node/npm versions
+Make sure you are using Node v6.2.2 and npm v3.9.5.
```

Comment:
I just check out of curiosity what I'm using and I have node v5.10.1 and npm 3.10.5 . Maybe the note needs to sound a less strict. "Make sure you are using recent versions of node and npm. No problem have been seen with `node>=5.10` and `npm>=3.9`"